### DISCUSSION TITLE
How skywalking supports multiple environments

### DISCUSSION THREAD

**Discussion by X-Zackt** [apache/skywalking#8126] [Q&A]
I have deployed a skywalking for my project.but my test environment is multipled. how can i use skywalking with multple environments?
I tryed to use agent.config with parameter 'agent.namespace',but can't effect.
how should i do ?

**Comment by wu-sheng** [comment]
Please search before submit discussion. This has been explained again and again.

**Reply by X-Zackt** [reply]
I have searched,buf can't find. can you give me a link?

**Reply by wu-sheng** [reply]
Exactly the same terms you are using here, https://github.com/apache/skywalking/issues/5344. I am not sure why you said you can't.

**Reply by X-Zackt** [reply]
thank you.Maybe I don't want to admit that SW doesn't support multiple environments.

**Reply by wu-sheng** [reply]
Every feature was contributed by people from community. If no one intends to work on that, then, it doesn't exist. Very simple logic.
